dill/_objects.py

```diff
@@ -111,6 +109,7 @@ class _Struct(ctypes.Structure):
         pass
     _Struct._fields_ = [("_field", ctypes.c_int),("next", ctypes.POINTER(_Struct))]
 _filedescrip, _tempfile = tempfile.mkstemp('r') # deleted in cleanup
+os.close(_filedescrip)
```

Comment:
Yes, I noticed that list night as I was debugging on windows. Therefore it needs to be handled as is for non-windows systems, and changed for windows systems.  This might also impact the tests below.  So… new PR.  It's a bit more to investigate than a simple pull.